src/connectors/paymentService.ts

```diff
@@ -1072,4 +1078,102 @@ export class PaymentService extends BaseService {
       })
       .returning('*')
   }
+
+  /*********************************
+   *                               *
+   *           notification        *
+   *                               *
+   *********************************/
+
+  notifyDonation = async ({
+    tx,
+    sender,
+    recipient,
+    article,
+  }: {
+    tx: Transaction
+    sender: User
+    recipient: User
+    article: {
+      title: string
+      slug: string
+      authorId: string
+      mediaHash: string
+      draftId: string
+    }
+  }) => {
+    const atomService = new AtomService()
+    const notificationService = new NotificationService()
+    const amount = parseFloat(tx.amount)
+    // send email to sender
+    const author = await atomService.findFirst({
+      table: 'user',
+      where: { id: article.authorId },
+    })
+    const draft = await atomService.findFirst({
+      table: 'draft',
+      where: { id: article.draftId },
+    })
+
+    const hasReplyToDonator =
```

Comment:
`const hasReplyToDonator = !!draft.replyToDonator` is identical.

Reply:
is `!!draft.replyToDonator` clear in the case of  empty string?

Reply:
Yeah it would be clear enough (for JS developer), lol.

Reply:
cool... I will fix it